### ISSUE TITLE
Create UI for settings/about section

### ISSUE THREAD

**Issue by kdzwinel** [kdzwinel/betwixt#34]
We need a space to put settings (e.g. port number, reverse proxy options) and other stuff (root certificate download button, link to github, license info, etc.). The plan is to show the tab bar back again and create a new, custom tab with Settings/About.

<img width="197" alt="screen shot 2016-03-15 at 11 01 55" src="https://cloud.githubusercontent.com/assets/985504/13774120/81896dc2-ea9d-11e5-9276-34e30ede893d.png">

**Comment by Garbee** [comment]
Why do a tab vs a settings icon in the action area that opens a dialog? That would save the app real-estate for the content.

**Comment by kdzwinel** [comment]
That's also an option - I'll see how easy it is to add that icon to the action area (in regular devtools it's on the tab bar). Other option is to trigger 'preferences' from the native menu.

<img width="233" alt="screen shot 2016-03-30 at 00 49 06" src="https://cloud.githubusercontent.com/assets/985504/14126433/3c896860-f611-11e5-9ae1-a30e90e05450.png">

**Comment by Garbee** [comment]
Native menu is only on MacOS. Linux doesn't have one for Betwixed and
Windows I'm not sure, but they don't have a standard OS menu like that.
Unless you want one thing for Mac then something else for Linux/Windows
then it doesn't seem like a good idea. Since doing native menus on
Windows/Linux then means removing more screen real-estate from the app,
which is the whole point in just doing an icon + dialog.

**Comment by kdzwinel** [comment]
Good point. I definitely don't want to take away too much space or do different versions for different OSes. As far as I tested native menu on windows appears inside Betwixt window so it's not ideal. I'll explore the cog/hamburger option as you suggest.
